extensions/database/module/scripts/index/database-parsing-panel.html

```diff
@@ -1,53 +1,119 @@
-<div bind="wizardHeader" class="database-importing-wizard-header">	
+<div bind="wizardHeader" class="database-importing-wizard-header">
   <div class="grid-layout layout-tightest layout-full">
     <table>
       <tr>
-        <td width="1%"><button bind="startOverButton" class="button">&laquo; Start Over</button></td>
+        <td width="1%">
+          <button bind="startOverButton" class="button">
+            &laquo; Start Over
+          </button>
+        </td>
```

Comment:
It would be good if you could avoid doing any reformatting, since it makes it much harder to review your changes. Typically, the change above does not contain any functional change.